eclair-core/src/main/resources/reference.conf

```diff
@@ -37,13 +37,11 @@ eclair {
   node-color = "49daaa"
 
   trampoline-payments-enable = false // TODO: @t-bast: once spec-ed this should use a global feature flag
-  global-features = "0200" // variable_length_onion
-  local-features = "088a" // initial_routing_sync + option_data_loss_protect + option_channel_range_queries + option_channel_range_queries_ex
+  features = "0a8a" // initial_routing_sync + option_data_loss_protect + option_channel_range_queries + option_channel_range_queries_ex + variable_length_onion
```

Comment:
We could enhance the [`StartupSpec`](https://github.com/ACINQ/eclair/blob/master/eclair-core/src/test/scala/fr/acinq/eclair/StartupSpec.scala#L31) configuration test to prevent people from using the old config keys.

Reply:
Great idea, I'll do that!